x/staketia/keeper/migration.go

```diff
@@ -0,0 +1,159 @@
+package keeper
+
+import (
+	"errors"
+
+	errorsmod "cosmossdk.io/errors"
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	recordtypes "github.com/Stride-Labs/stride/v22/x/records/types"
+	stakeibctypes "github.com/Stride-Labs/stride/v22/x/stakeibc/types"
+	oldtypes "github.com/Stride-Labs/stride/v22/x/staketia/legacytypes"
+	"github.com/Stride-Labs/stride/v22/x/staketia/types"
+)
+
+// TODO [UPGRADE HANDLER]: Migrate stakeibc host zone (set redemptions enabled to true on each host zone)
```

Comment:
Good flat - should we add this in here or is it tracking in a separate PR?

Reply:
I believe it was handled separately. Updated code is [here](https://github.com/Stride-Labs/stride/pull/1223/files#diff-8a0bb7f321cfb197fe6e15baf17d0db0ecd70e00f4d983d726f7764b1f620a19)